src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+* {
+  margin: 0;
```

Comment:
better to avoid using `*` selector for margins reseting (not every html has padding, so better to specify particular tags to reset)